data/scannet/batch_load_scannet_data.py

```diff
@@ -77,19 +70,15 @@ def batch_export(max_num_point, output_folder, train_scan_names_file,
             print('-' * 20 + 'done')
             continue
         try:
-            export_one_scan(scan_name, output_filename_prefix, max_num_point,
-                            label_map_file, scannet_dir)
+            export_one_scan(scan_name, output_filename_prefix, label_map_file,
+                            scannet_dir)
         except Exception:
             print(f'Failed export scan: {scan_name}')
         print('-' * 20 + 'done')
 
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument(
-        '--max_num_point',
-        default=50000,
-        help='The maximum number of the points.')
```

Comment:
I am not sure. But I don't think it's necessary (and reasonable) to sub-sample points in this step. After all this will distort the original data. Maybe ask @ZwwWayne here.

Reply:
May compare the efficiency of both options or list some numbers for further consideration. For example, the number of original points and the number of points need to be used in seg/det tasks.

Reply:
For the detection task, 50000 points might be enough, while for the segmentation task, we need to output a label for EVERY point in the raw data. I add an option to allow user to determine whether generate ScanNet data for seg/det task.